dsgrid/cli/query.py

```diff
@@ -387,7 +391,13 @@ def query_composite_dataset(
     # CompositeDatasetQuerySubmitter.submit(project, output).submit(query, force=force)
 
 
-@click.command()
+_create_derived_dataset_config_epilog = """
+Examples:\n
+$ dsgrid query project create-derived-dataset-config query_output/my_result my_dataset_config\n
+"""
```

Comment:
I'm not sure what `my_dataset_config` is ... is it a name, a folder, a file name?

`my_result` is also slightly unclear ... should it be `my-query-result-name` (from `_create_project_query_epilog`)?

Reply:
The command doctoring is `Create a derived dataset configuration and dimensions from a query result.` `my_dataset_config` will be a directory that contains a dataset.json5 as well as possibly some dimension files.